giskard/commands/cli_worker.py

```diff
@@ -160,9 +160,11 @@ def _start_command(is_server, url: AnyHttpUrl, api_key, is_daemon, hf_token=None
 
             run_daemon(is_server, url, api_key, hf_token)
         else:
-            if sys.platform == "win32":
+            if settings.force_asyncio_event_loop or sys.platform == "win32":
```

Comment:
I have issues when debugging worker from intellij when uvloop is used. I added the flag mostly for myself, but in case the same problems appear in prod there'll be a way to let people switch from one implementation to the other